packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -104,13 +113,25 @@ export class PeoplePage {
 	 * @param {string} emailAddress Email address of the pending user.
 	 */
 	async selectInvitedUser( emailAddress: string ): Promise< void > {
-		await this.page.click( selectors.invitedUser( emailAddress ) );
+		await this.waitUntilLoaded();
+
+		// Ensure the card for the invited user is stable and loaded before clicking,
+		// otherwise the click is ignored.
+		const elementHandle = await this.page.waitForSelector( selectors.invitedUser( emailAddress ) );
+		await elementHandle.waitForElementState( 'stable' );
```

Comment:
I don't think this might be having the direct intended effect - if you look at the [auto-waiting tables here](https://playwright.dev/docs/actionability/), click will already wait for the element to be stable.

I think what this is doing more than anything is just adding some milliseconds up front that lets this button settle.

That means what might be going on here is that the element is there, but the handlers might not do anything yet (kind of like the issue with the Close Account button), so we may need to find something else to key off of on the page to know that all the handlers have been set up correctly.

Reply:
I think you're totally right on that - but on the other hand, this passed stress test multiple times, so maybe it's the little bit of extra wait that's necessary.

I'll try to see what else I can wait on.

Reply:
I've pushed up a new revision and put it under stress test - the new revision waits on the HTTP request for `invites?` to receive a response instead.

[desktop](https://teamcity.a8c.com/buildConfiguration/calypso_Calypso_E2E_Playwright_desktop/6779934?buildTab=log&focusLine=273)
[mobile](https://teamcity.a8c.com/buildConfiguration/calypso_Calypso_E2E_Playwright_mobile/6779941)

Reply:
Stress test fails but I think that's because of the spam-like amount of invites being sent out.

Reply:
Sure, I think that's a good place to start, and we can see how it pans out over time and adjust as needed! We keep running into this issue in Claypso - things being "enabeld" but not really, so we'll have to keep trying out things to see what works best and most smoothly!

Also, I echo what @WunderBart says [below](https://github.com/Automattic/wp-calypso/pull/56474#issuecomment-932072838) - I think you may have found a bug there! Our invites may not handle stress testing!!

Reply:
@dpasque @WunderBart Good point about the banner, I had written it off as just another inconsistency but that's kind of defeating the point of our work, isn't it 😄 

I'll file a new issue for this one!

Reply:
Filed https://github.com/Automattic/wp-calypso/issues/56758.